kits/java/localrun/main.py

```diff
@@ -0,0 +1,116 @@
+import json
```

Comment:
Do we need this file? Does it work locally?

Reply:
I think you should just be able to add
```json
".jar": "java -jar"
```
to [luxai_runner/ext_to_command.py](https://github.com/Lux-AI-Challenge/Lux-Design-2022/blob/main/luxai_runner/ext_to_command.py). Then you can run your bot with the local runner like
```bash
luxai2022 path/to/JavaBot.jar path/to_another/JavaBot.jar -o replay.json
```

Reply:
Yes, question of adding ".jar": "java -jar" was discussed, but a mediator main.py was created.